build.gradle

```diff
@@ -20,6 +20,10 @@ import java.time.format.DateTimeFormatter
 
 plugins {
   id 'base'
+  alias(libs.plugins.kotlinMultiplatform) apply false
```

Comment:
I added the plugins in the project gradle so that they are loaded once (currently used by two modules with different versions).

These changes are not necessary as such and came from a draft I made before but forgot to clean up. The draft was implementing a Solr client for Kotlin for generating classes from OAS, and I had to sync the OpenAPI plugin versions.

If I remember correct, the plugin for Kotlin multiplatform and Compose should be loaded at project level (haven't tried it at module level) and only applied to modules were necessary.

The multiplatform plugin is used for supporting individual targets in the gradle module level (hence the `commonMain`, `jvmMain`, `wasmJsMain` sub-modules / directories in the new ui module). This way, we can provide platform specific code if necessary (see [expect-actual declarations](https://kotlinlang.org/docs/multiplatform-expect-actual.html)).